README.md

```diff
@@ -54,11 +54,68 @@
+
+According to a recent [Sysdig study](https://sysdig.com/blog/millions-wasted-kubernetes/), on average, Kubernetes clusters have:
+
+-   69% unused CPU
+-   18% unused memory
+
+By utilizing Robusta KRR's recommendations, you can significantly reduce these inefficiencies.
+
+### How it works
+
+#### Metrics Gathering
+
+Robusta KRR uses the following Prometheus queries to gather usage data:
+
+-   CPU Usage:
+
+    ```
+    sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{namespace="{object.namespace}", pod="{pod}", container="{object.container}"})
+    ```
+
+-   Memory Usage:
+
+    ```
+    sum(container_memory_working_set_bytes{job="kubelet", metrics_path="/metrics/cadvisor", image!="", namespace="{object.namespace}", pod="{pod}", container="{object.container}"})
+    ```
+
+[_Need to customize the metrics? Tell us and we'll add support._](https://github.com/robusta-dev/robusta-krr/issues/new)
+
+#### Resource Recommendations
+
+By default, we use a _simple_ strategy to calculate resource recommendations. It is calculated as follows (_The exact numbers can be customized in CLI arguments_):
+
+-   For CPU, we set a request at the 99th percentile with no limit. Meaning, in 99% of the cases, your CPU request will be sufficient. For the remaining 1%, we set no limit. This means your pod can burst and use any CPU available on the node - e.g. CPU that other pods requested but aren’t using right now.
+
+-   For memory, we take the maximum value over the past week and add a 5% buffer.
+
+### Difference with Kubernetes VPA
+
+| Feature 🛠️                  | Robusta KRR 🚀                                                                                             | Kubernetes VPA 🌐                                           |
+| --------------------------- | ---------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| Resource Recommendations 💡 | ✅ CPU/Memory requests and limits                                                                          | ✅ CPU/Memory requests and limits                           |
+| Installation Location 🌍    | ✅ Not required to be installed inside the cluster, can be used on your own device, connected to a cluster | ❌ Must be installed inside the cluster                     |
+| Workload Configuration 🔧   | ✅ No need to configure a VPA object for each workload                                                     | ❌ Requires VPA object configuration for each workload      |
+| Immediate Results ⚡        | ✅ Gets results immediately (given Prometheus is running)                                                  | ❌ Requires time to gather data and provide recommendations |
+| Extensibility 🔧            | ✅ Supports multiple recommendation strategies, extendable                                                 | ❌ Limited extensibility                                    |
```

Comment:
In theory VPA can be extended too. I would write here "Add your own strategies with 5 lines of Python" or something like that.